src/Fixer/LanguageConstruct/DeclareBracesFixer.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Fixer\LanguageConstruct;
+
+use PhpCsFixer\AbstractFixer;
+use PhpCsFixer\FixerDefinition\CodeSample;
+use PhpCsFixer\FixerDefinition\FixerDefinition;
+use PhpCsFixer\FixerDefinition\FixerDefinitionInterface;
+use PhpCsFixer\Tokenizer\Tokens;
+
+final class DeclareBracesFixer extends AbstractFixer
+{
+    public function getDefinition(): FixerDefinitionInterface
+    {
+        return new FixerDefinition(
+            'There must not be spaces around `declare` statement braces.',
```

Comment:
`around braces` or `inside parenthesis` ?

Reply:
to my understanding, `braces` means `{`/`}`
in PHP CS Fixer, for `(` / `)` we were using parentheses.

i will late myself apply the rename

Reply:
> to my understanding, `braces` means `{`/`}`
> in PHP CS Fixer, for `(` / `)` we were using parentheses.

if we were that consistent - we have curly braces even in some fixer name

Reply:
@kubawerlos https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/5915